src/components/header.tsx

```diff
@@ -1,11 +1,15 @@
 import { Link } from 'waku';
+import { JoinDiscordServer } from './joinDiscordServer';
 
 export const Header = () => {
   return (
-    <header className="flex items-center gap-4 p-6 lg:fixed lg:left-0 lg:top-0">
-      <h2 className="text-2xl font-bold tracking-tight">
+    <header className="flex items-center gap-4 p-6 w-full lg:fixed lg:left-0 lg:top-0">
+      <h2 className="text-2xl font-bold tracking-tight flex-grow">
         <Link to="/">React Tokyo</Link>
       </h2>
+      <div className="ml-auto">
+        <JoinDiscordServer />
+      </div>
```

Comment:
意図を読み取れてなかったらすみません。こちらの方がシンプルかなと思ったのですが、いかがでしょうか？

```suggestion
    <header className="flex items-center p-6 justify-between w-full lg:fixed lg:left-0 lg:top-0">
      <h2 className="text-2xl font-bold tracking-tight">
        <Link to="/">React Tokyo</Link>
      </h2>
      <div>
        <JoinDiscordServer />
      </div>
```

Reply:
@aTakatoNakamura 
ご対応ありがとうございます。

サイズ調整良いと思います！391pxは細かすぎましたね🙇

Reply:
@dai-shi 
私は良いと思うのですがいかがでしょうか？